src/components/BlogFeed.astro

```diff
@@ -67,8 +83,14 @@ type ArticleResult = {
             <img src={image.src} width={image.width} alt={media.data.alt} />
           </figure>
           <h4>{post.webTitle}</h4>
+          </a>
+          <h5 class="byline">
+          {post.tags[0] ? 
+            (<a href={post.tags[0].webUrl} rel="author">{post.tags[0].webTitle}</a>) 
+            : post.fields.byline}
+          </h5>
           <h5>{pubDate}</h5>
-        </a>
```

Comment:
I really like the idea of adding a byline, but I’m not sure it’s ideal to prevent the whole card being a link to the article.

On fronts, usually the whole card is clickable, and you cannot get to an author’s page without going to the article first. I think that’s a behaviour we’d want to keep.

Reply:
I tend to agree with you. The main issue here is that according to [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel), the `rel` attribute is only valid for a small set of HTML tags (namely `<a>`, `<area>`, `<link>` and `<form>`).

Adding `rel="author"` to an `<a>` tag that links to a blog post feels semantically incorrect to me, which is why I added an extra link. This has the disadvantage of being clunky (as a user, if I click on a card I expect to be taken to the blog post regardless of which bit of text/image I click on).

I'd be inclined to remove the `rel="author“` attribute altogether, restore original behaviour, and restrict the scope of the changes to the PR to just removing the `<em>` tag from the byline and moving the italic style to the Astro `<style>` tag.  

Refactoring the HTML tags used in the card should happen in a separate PR, imho.

Reply:
Great, I’m all for removing the `rel="author"` at this point. Maybe we can revisit further, but hopefully it was a good experimentations with CAPI :)